source/auth/auth.rst

```diff
@@ -1213,14 +1221,113 @@ mechanism
 mechanism_properties
     PROVIDER_NAME
         Drivers MUST allow the user to specify the name of the OIDC service to
-        use to obtain credentials. MUST be one of ["aws", "custom"].
+        use to obtain credentials. MUST be one of ["aws"].
```

Comment:
Is custom callback usage just the expected default here, if it can no longer be specified in the `PROVIDER_NAME` field?

Reply:
It's not the expected default. However, having to specify `PROVIDER_NAME="custom"` as well as specifying a custom callback seems redundant. Instead, I've updated the document to say that `PROVIDER_NAME` and a custom callback cannot be used together and drivers should return an error if both are specified.

Reply:
Yeah, that clarifies things. Thank you!